openff/toolkit/topology/molecule.py

```diff
@@ -6470,3 +6410,57 @@ def _ipython_display_(self):
             return display(self.visualize(backend="openeye"))
         except ValueError:
             pass
+
+
+@requires_package("networkx")
```

Comment:
(discussed in our meeting today) NetworkX is a hard dep of the toolkit, so we don't need to check for it here.

Reply:
For context: it's lazy-loaded elsewhere in the module, which I confused with being optional